crates/block-producer/src/mempool/tests.rs

```diff
@@ -7,11 +8,11 @@ use crate::test_utils::MockProvenTxBuilder;
 impl Mempool {
     fn for_tests() -> Self {
         Self::new(
-            BlockNumber::new(0),
+            BlockNumber::from(0),
```

Comment:
I believe there is also `BlockNumber::default()` or `BlockNumber::GENESIS`

Reply:
I had missed `BlockNumber::GENESIS`, but default looked a bit opaque which is why I changed it to this originally.